modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/util/LabelItem.java

```diff
@@ -40,26 +42,50 @@ public void putData(String key, String value) {
 		data.put(key, value);
 	}
 
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), replaced by {@link
+	 *             #setDismissible()}
+	 */
+	@Deprecated
 	public void setCloseable(boolean closeable) {
 		put("closeable", closeable);
+		setDismissible(closeable);
 	}
 
 	public void setData(Map<String, Object> data) {
 		put("data", data);
 	}
 
+	public void setDismissible(boolean dismissible) {
+		put("dismissible", dismissible);
+	}
+
+	public void setDisplayType(String displayType) {
+		put("displayType", displayType);
+	}
+
 	public void setLabel(String label) {
 		put("label", label);
 	}
 
+	public void setLarge(boolean large) {
+		put("large", large);
+	}
+
 	public void setStatus(int status) {
 		setLabel(WorkflowConstants.getStatusLabel(status));
 
 		setStyle(WorkflowConstants.getStatusStyle(status));
 	}
 
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), replaced by {@link
+	 *             #setDisplayType()}
+	 */
+	@Deprecated
 	public void setStyle(String style) {
 		put("style", style);
```

Comment:
Same as with `closeable`. Maybe remove `style` and always depend on `displayType`. Otherwise, we are providing inconsistent APIs.

Reply:
true too too

Reply:
Not doing it because https://github.com/liferay-frontend/liferay-portal/pull/434#discussion_r509314754